apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/JobCreationOptions.cs

```diff
@@ -21,6 +22,8 @@ namespace Google.Cloud.BigQuery.V2
     /// </summary>
     public abstract class JobCreationOptions
     {
+        private const string SessionIdKey = "session_id";
```

Comment:
Should the changes *actually* apply to all job creation options? We're not doing anything with them for CreateCopyJobOptions, CreateExtractJobOptions etc.

If they're only relevant for CreateLoadJobOptions and QueryOptions, they should only be there. My earlier comments about avoiding duplication may have been a red herring here. Happy to chat on a VC.